core/ir/aarch64/codec.c

```diff
@@ -1809,6 +1822,21 @@ encode_opnd_sysops(uint enc, int opcode, byte *pc, opnd_t opnd, OUT uint *enc_ou
     return encode_opnd_int(5, 14, false, 0, 0, opnd, enc_out);
 }
 
+/* wx5_imm5: bits 5-9 is a GPR whos width is dependent on information in
```

Comment:
This comment looks like a copy-and-paste error from `wx5_imm5`.

Reply:
done